src/pps/pps.proto

```diff
@@ -1052,6 +1053,19 @@ message PipelinesSummary {
   int64 unhealthy_pipelines = 5;
 }
 
+// PipelinePicker is used to select a single pipeline.  It is a pfs.RepoPicker
+// where the repo type defaults to "spec."
+
+message PipelinePicker {
+  message RepoName {
+    pfs_v2.ProjectPicker project = 1;
+    string name = 2;
```

Comment:
Should we have a validation rule to require this to be non-empty?

Reply:
The PFS Repo Picker message this copies doesn’t have a validation rule. Maybe it should? OTOH, an invalid picker will get an error when actually used.